lib/when_committed.rb

```diff
@@ -24,7 +24,7 @@ def initialize(&callback)
       @callback = callback
     end
 
-    def committed!(should_run_callbacks=true)
+    def committed!(should_run_callbacks: true)
```

Comment:
I think it is worth adding a wildcard to collect (and ignore) any other keyword args that may be passed `(should_run_callbacks: true, **)`